Smartling.API/Model/Account.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace Smartling.Api.Model
+{
+    public class Account
+    {
+        [JsonProperty("accountName")]
```

Comment:
All these similar attributes are annoying. Just configure json lib to use camelCase strategy. It's a common feature in all frameworks.

Reply:
Named in accordance with a codestyle used in the project.